cmd/aws-janitor-boskos/main.go

```diff
@@ -74,22 +85,25 @@ func main() {
 }
 
 func run(boskos *client.Client) error {
+	bTypes := strings.Split(rTypes.String(), ",")
```

Comment:
I think you don't need this `Split` - you can just iterate over rTypes directly (internally it's a slice)